jobs/innovint_contract_django.html

```diff
@@ -0,0 +1,34 @@
+---
+title: Django Backend Contract Project
+company: InnoVint
+url: www.innovint.us
+location: SF bay area & Portland (Remote OK!)
+contract: contract
+contact:
+    name: Dan Strengier
+    email: dan@innovint.us
+created: !!timestamp '2016-05-18'
+tags:
+  - python
+  - dango
+  - postgress 
+---
+
+#Project Description
+
+Our client stores lab-analysis data on an FTP server for over 3000 customers, but those customers have no ability to access this digital data. 
+We are building a web application that allows customers to log in and view their lab analyses.
+We have finished designing & building the front end Angular app. We made significant progress on a Scala backend, 
+but decided to scrap it in favor for a simpler Python + Django backend solution. 
+We are looking for a developer to build this Python back end while working closely with the project manager and front end developer.
+
+#Project Requirements
+This contract project requires a Python/Django app to be built with the following goals:
+--Grab lab-analysis data from an external FTP server, then parse and save this data in our Postgres DB. 
+--Create REST APIs to access the saved data in Postgres, which our Angular App will consume. 
+--Utilize some user-management solution that allows clients to access their unique data, reset their password, add/remove users from teams, etc. Ideally, an existing framework is used for this functionality, as we don’t want to build a user management system from scratch.
+We have clear functional specs, a Postgres schema defined and implemented, APIs already defined, and a solid Angular app built. This should be relatively straightforward effort.
+
+You will be the sole backend engineer on this project. We need someone with strong abilities to execute. 
+We will need a full proposal on level of effort, time estimations, and agreed upon milestones.
+Remote is fine. We can meet in person in the SF bay area & Portland (if you happen to be in either locations).
```

Comment:
Same again here - there's no reason to double-up your ads (since they are free). You can just put in one ad for each location where you are trying to attract candidates.